test/integration/hotrestart_test.sh

```diff
@@ -152,7 +152,8 @@ function run_testsuite() {
   # To ensure that we don't accidentally change the /hot_restart_version
   # string, compare it against a hard-coded string.
   start_test "Checking for consistency of /hot_restart_version"
-  CLI_HOT_RESTART_VERSION=$("${ENVOY_BIN}" --hot-restart-version --base-id "${BASE_ID}" 2>&1)
+  # Get last line to avoid the influence of some bootstrap log.
+  CLI_HOT_RESTART_VERSION=$("${ENVOY_BIN}" --hot-restart-version --base-id "${BASE_ID}" 2>&1 | sed -n '$p')
```

Comment:
Just out of curiosity why is this change needed as part of this PR?

/wait-any

Reply:
The tcmalloc added a new bootstrap log which will pollute the output of ` --hot-restart-version`.

Reply:
Hmm, this seems like a problem, as this will effect other people that have scripts. Are they printing something to stderr directly? I think we need to disable this somehow. What is the log exactly? cc @ckennelly 

/wait

Reply:
Here is the log.
```
$ envoy --hot-restart-version
external/com_github_google_tcmalloc/tcmalloc/internal/percpu_tcmalloc.h:1271] Bytes used per cpu of available 1179936 1572864
11.104
```

Reply:
> Are they printing something to stderr directly?

Yeah. Here are some related code snippets. I don't aware any switch here.

```
// Log function.
void Log(LogMode mode, const char* filename, int line, LogItem a, LogItem b,
         LogItem c, LogItem d, LogItem e, LogItem f) {
  Logger state =
      FormatLog(mode == kLogWithStack, filename, line, a, b, c, d, e, f);
  int msglen = state.p_ - state.buf_;
  (*log_message_writer)(state.buf_, msglen);
}

// Writer.
static void WriteMessage(const char* msg, int length) {
  (void)::write(STDERR_FILENO, msg, length);
}
void (*log_message_writer)(const char* msg, int length) = WriteMessage;
```

Reply:
IMO we have to do a patch here to allow this to be disabled/configured.

Reply:
> IMO we have to do a patch here to allow this to be disabled/configured.

Do we need any log of tcmalloc? If a global switch can be accepted, may be a simple macro is enough. Or need we to enhance the log system of tcmalloc to support multiple log level?

cc @ckennelly

Reply:
> We should just delete that log message. It's not super informative, and not really actionable for users.
> 
> I'll aim to land the change tomorrow to delete it.

Great, thanks.